docs/source/index.rst

```diff
@@ -0,0 +1,67 @@
+.. title::
+   QiboTN
+
+What is QiboTN?
+==================================
+QiboTN  is the dedicated `Qibo <https://github.com/qiboteam/qibo>`_ backend to support large-scale simulation of quantum circuits and acceleration.
+
+Supported Computation:
+
+- Tensornet (TN)
+- Matrix Product States (MPS)
+
+Tensor Network contractions to:
+
+- dense vectors
+- expecation values of given Pauli string
+
+The supported HPC configurations are:
+
+- single-node CPU
+- single-node GPU or GPUs
+- multi-node multi-GPU with Message Passing Interface (MPI)
+- multi-node multi-GPU with NVIDIA Collective Communications Library (NCCL)
+
+Currently, the supported tensor network libraries are:
+
+- [cuQuantum](https://github.com/NVIDIA/cuQuantum), an NVIDIA SDK of optimized libraries and tools for accelerating quantum computing workflows.
+- [quimb](https://quimb.readthedocs.io/en/latest/), an easy but fast python library for ‘quantum information many-body’ calculations, focusing primarily on tensor networks.
+
+How to Use the Documentation
+============================
+
+Welcome to the comprehensive documentation for QiboTN! This guide will help you navigate through the various sections and make the most of the resources available.
+
+
+1. **Getting started**: Begin by referring to the
+   :doc:`/Installation/` guide to set up the ``QiboTN`` library in your environment.
+
+2. **Tutorials**: Explore the :doc:`Quickstart/` section for basic usage examples
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+
+Contents
+--------
+--------
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Introduction
+   usage
+   api
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Main documentation
+
+    api-reference/modules
```

Comment:
We need to put in the right content based on the basic source files available now.